modules/UI/videolayout/SmallVideo.js

```diff
@@ -557,24 +557,19 @@ SmallVideo.prototype._isHovered = function() {
  * Hides or shows the user's avatar.
  * This update assumes that large video had been updated and we will
  * reflect it on this small video.
- *
- * @param show whether we should show the avatar or not
- * video because there is no dominant speaker and no focused speaker
  */
 SmallVideo.prototype.updateView = function() {
     if (this.disableUpdateView) {
         return;
     }
 
-    if (!this.hasAvatar) {
```

Comment:
@saghul It seems `this.hasAvatar` is not used anymore but has a few setters here and there. Am I correct?

Reply:
It’s possible this was the last use. Sorry I didn’t notice!